test/MorseL.Client.WebSockets.Tests/MorseL.Client.WebSockets.Tests.csproj

```diff
@@ -5,18 +5,14 @@
   </PropertyGroup>
 
   <ItemGroup>
-      <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0" />
-    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.3.0" />
-    <PackageReference Include="xunit" Version="2.2.0" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.2.0" />
+    <PackageReference Include="Microsoft.AspNetCore.All" Version="2.0.0" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="15.9.0" />
+    <PackageReference Include="xunit" Version="2.4.0" />
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.0" />
   </ItemGroup>
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\MorseL.Client.WebSockets\MorseL.Client.WebSockets.csproj" />
   </ItemGroup>
 
-  <ItemGroup>
```

Comment:
Hm, I wonder what this was for?

Reply:
Not sure. Cleaning up the files and updating the dependencies resulted in removing some odd "multiple test adapters found for uri" messages and accurate detection of tests. If you look at the old test results you'll see a lot of shenanigans getting the tests to actually run before these modifications were made.